### DISCUSSION TITLE
Migrate maps DateTime to MySQL datetime, not timestamp?

### DISCUSSION THREAD

**Discussion by techntrails** [prisma/prisma#3581] [Q&A] [answered]
Hi all,

The docs day that a prisma DateTime should migrate to a MySQL timestamp. But for me it's creating a MySQL **datetime**. Any ideas please?

https://www.prisma.io/docs/reference/database-connectors/mysql#type-mapping-between-mysql-to-prisma-schema

**Comment by ryands17** [comment]
@techntrails 

> The docs day that a prisma DateTime should migrate to a MySQL timestamp. But for me it's creating a MySQL timestamp. Any ideas please?

The thing that you've written above is the same, so could you clarify?

**Reply by techntrails** [reply]
My mistake. I've just updated original post. Changes in bold. Thanks Ryan.

**Reply by ryands17** [reply, marked as answer]
Hey @techntrails 👋 
Thanks a lot for calling this out! We have updated and the changes have been reflected in the docs :)

**Reply by techntrails** [reply]
Good to know, thank you. This was doing my head in. I even upgraded from a March beta release to 2.6.2 thinking the mappings might have changed.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
